### PR TITLE
Correcting typo

### PR DESCRIPTION
**Pull request by sandoche**

### CODE REVIEW

**Review by MaryJaneLove4 — APPROVED** [review]
Ano

**Review by MaryJaneLove4 — APPROVED** [review]

**Comment by MaryJaneLove4** [comment]
ok

**Review by katopz — APPROVED** [review]

**Review by katopz — APPROVED** [review]
LGTM

**Review by huangsuyu — APPROVED** [review]

File: docs/smart-chain/developer/issue-BEP20.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ d. Step4: Click “ABI” to copy the contract abi and save it.
 
 <img src="https://lh6.googleusercontent.com/qY_5g3ZMnJca6n84W2JxIoBvd8iHRQ0qkOQuJ60pRIcKvgZB5-bXcGq6gS7dFwA5rYXbiS2NyaUQ1Qptcagqa0pb7kmq_S-Dh8drA4R-hDg9_NVp1zPl-tmqIDanlcgLibaR3CV7" alt="img" style="zoom:50%;" />
 
-6. Depoy the contract to BSC
```
Comment (MaryJaneLove4) [inline docs/smart-chain/developer/issue-BEP20.md:30]: Deploy

**Review by MaryJaneLove4 — COMMENTED** [review]
sandoche:patch-1

**Pull request merged**